MODULE.bazel

```diff
@@ -50,6 +51,51 @@ node.toolchain(
 )
 use_repo(node, "nodejs", "nodejs_toolchains")
 
+kotlin_extractor_deps = use_extension("//java/kotlin-extractor:deps.bzl", "kotlin_extractor_deps")
+
+# following list can be kept in sync by running `bazel mod tidy` in `codeql`
```

Comment:
yeah, the build will fail. When building from `codeql` there will also be a helpful message suggesting to run `bazel mod tidy`, but I'm not so sure about building from `semmle-code`. In any case I intend to add a CI check for building from `codeql`. It might be that the error message when building from `semmle-code` in the QLucie check is a bit less clear.

Reply:
I've added a build check. Here's a test PR check run where that list was not in sync: https://github.com/github/codeql/actions/runs/8659912532/job/23746747549?pr=16193

Here's an excerpt of the logs:
```
WARNING: /home/runner/work/codeql/codeql/MODULE.bazel:54:38: The module extension kotlin_extractor_deps defined in @codeql//java/kotlin-extractor:deps.bzl reported incorrect imports of repositories via use_repo():

Imported, but not created by the extension (will cause the build to fail):
    kotlin-compiler-1.6.0, kotlin-compiler-embeddable-1.6.0, kotlin-stdlib-1.6.0

Fix the use_repo calls by running 'bazel mod tidy'.
ERROR: Target parsing failed due to unexpected exception: module extension "kotlin_extractor_deps" from "//java/kotlin-extractor:deps.bzl" does not generate repository "kotlin-compiler-1.6.0", yet it is imported as "kotlin-compiler-1.6.0" in the usage at /home/runner/work/codeql/codeql/MODULE.bazel:54:38 (did you mean 'kotlin-compiler-1.6.[20](https://github.com/github/codeql/actions/runs/8659912532/job/23746747549?pr=16193#step:3:21)'?)
```

Reply:
This check won't be required by the PR, as the "official" build will be regularly checked by QLucie, but will provide faster and possibly clearer feedback in case of build problems